pkg/config/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package config manages CLI configurations for your DC/OS clusters.
+//
+// Each configured cluster has an associated TOML configuration file which can be parsed,
```

Comment:
Let's cut the comment just after `file` and bring the rest in the next line to have something of the same length.